spec/openapi.yaml

```diff
@@ -1,44 +1,147 @@
 openapi: 3.0.1
+
+servers:
+- url: https://resources.operationcode.org/api/v1
+  description: Default Server
 info:
   title: Operation Code Resources API
-  description: A collection of programming and cyber security learning resources for
-    the Operation Code community.
+  version: '1.0'
+  x-logo:
+    url: 'https://operation-code-assets.s3.us-east-2.amazonaws.com/branding/logos/small-blue-logo.png'
+    altText: 'Operation Code Logo'
+    backgroundColor: '#252e3e'
+  description: |
+    # Introduction
+    Operation Code Resources API is a collection of programming and cyber security learning resources curated by and for the Operation Code community.
+
+    # Authentication
+    Create (POST) and update (PUT) routes are protected and require an API key. API key requests to the `/apikey` endpoint require valid Operation Code credentials in the request body. 
+    
+    If you are not already a member of our community, join Operation Code [HERE](https://operationcode.org/join).
+
+    When you create a request to an authenticated route, you *MUST* include the header `x-apikey` along with your API key.
+
   contact:
     name: Operation Code
     url: https://operationcode.org/contact
-  version: "0.1"
-servers:
-- url: https://resources.operationcode.org/api/v1
+
 paths:
+  /apikey:
+    post:
+      summary: API Key Request
+      description: Request API key by providing valid Operation Code credentials
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ApiKeyRequest'
+        required: true
+      responses:
+        200:
+          description: API Key Response
```

Comment:
There's an inconsistency where some of these descriptions (particularly this one and the `4xx` descriptions) are capitalized, but other descriptions (such as `A list of resources` are not. I think that keeping the conversational tone is fine, let's make this `API key response` and all of the `404`s should be `Resource not found` and similar. The first letter of the first word should be capitalized and all of the rest of the should be lowercase